stablehlo/dialect/StablehloOps.td

```diff
@@ -770,7 +770,7 @@ def StableHLO_MulOp : StableHLO_BinaryElementwiseOp<"multiply",
     Returns `lhs * rhs` element-wise.
```

Comment:
Following [the example set by ConcatenateOp](https://github.com/openxla/stablehlo/blob/3bff283a7476489d0fea2fffad0e9bdec3587950/stablehlo/dialect/StablehloOps.td#L1800-L1810), let's update this sentence to be the same as the first sentence of the spec. Also see #250.